tests/budgeting/test_proposals_api_filtering.py

```diff
@@ -195,8 +200,10 @@ def test_proposal_list_filter_mixin(
 
 @pytest.mark.django_db
 def test_proposal_category_filter(
-    apiclient, module, proposal_factory, category_factory
+    apiclient, module, proposal_factory, category_factory, phase_factory
 ):
+    phase_factory(phase_content=phases.SupportPhase(), module=module)
+
```

Comment:
had to add a phase everywhere, because module.module_has_finished (used in is_allowed_view_vote_count) does not work with empty phase set